### PR TITLE
Fix kernel panic during tracing on Centos 7

### PR DESCRIPTION
**Pull request by mmkayPL**
Fixes issue #142

Signed-off-by: Mateusz Kozlowski <mateusz.kozlowski@intel.com>

### CODE REVIEW

File: source/kernel/io_trace.c [hunk 1]
```diff
@@ -122,12 +122,18 @@ static void bio_complete_event(void *ignore,
...
     unsigned cpu = get_cpu();
     struct iotrace_context *iotrace = iotrace_get_context();
 
     if (iotrace_bdev_is_added(&iotrace->bdev, cpu, q)) {
-        dev_id = disk_devt(IOTRACE_BIO_GET_DEV(bio));
+        disk = IOTRACE_BIO_GET_DEV(bio);
+        if (disk) {
+            dev_id = disk_devt(disk);
+        } else {
+            dev_id = (uint64_t) q->dev;
```
Comment (mmkayPL) [inline source/kernel/io_trace.c]: The trace could also be dropped, similar to what's done in RH:
https://www.redhat.com/archives/dm-devel/2013-January/msg00144.html

**Comment by tomaszrybicki** [comment]
It would be good to run some functional tests on this patch to verify tracing works properly

**Comment by tomaszrybicki** [comment]
retest this please

**Comment by tomaszrybicki** [comment]
retest this please

**Pull request merged**